src/Lecture1.hs

```diff
@@ -67,7 +70,10 @@ sumOfSquares x y = error "TODO!"
 
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n
+        | n > 0    = mod n 10
+        |otherwise = mod (-n) 10
```

Comment:
That's correct! 💯 
You can simplify the implementation a bit using the standard function `abs` that returns an absolute value of a number:

```haskell
λ: abs 7
7
λ: abs (-7)
7
```

Reply:
Yeah, makes sense. Don't know why I haven't thought of this. 🤦